src/main/java/cloud/fogbow/ras/core/plugins/interoperability/googlecloud/attachment/v1/GoogleCloudAttachmentPlugin.java

```diff
@@ -117,7 +118,7 @@ AttachmentInstance buildAttachmentInstanceFrom(GetAttachmentResponse response) {
         String device = response.getDevice();
 
         // TODO - Can I use 'status' from Cloud response?
-        String googleCloudState = EMPTY_STRING;
+        String googleCloudState = GoogleCloudConstants.EMPTY_STRING;
```

Comment:
Not only you can, but you should. This field is called cloudState. It should not be confused with OrderState.